__tests__/hooks/useGroupSetting.test.js

```diff
@@ -210,10 +216,10 @@ describe('use groupSetting should run correctly', () => {
         username: 'agita'
       }
     };
-    const spyToast = jest.spyOn(SimpleToast, 'show');
+    const spySetting = jest.spyOn(Linking, 'openSettings');
     const {result} = renderHook(() => useGroupSetting({navigation, route}), {wrapper});
     await result.current.lounchGalery();
-    expect(spyToast).toHaveBeenCalled();
+    expect(spySetting).toHaveBeenCalled();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The spy on `Linking.openSettings` has replaced the spy on `SimpleToast.show`. If the intention is to test whether the settings are opened when the gallery is launched, this change is appropriate. However, if there was a need to check if a toast message was shown, this change might not be suitable.
```diff
- const spyToast = jest.spyOn(SimpleToast, 'show');
+ const spySetting = jest.spyOn(Linking, 'openSettings');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->